catsoop/loader.py

```diff
@@ -225,7 +227,11 @@ def load_global_data(into, check_values=True):
         into["csm_base_context"] = into["base_context"] = base_context
         clean_builtins(into)
         into["csm_loader"] = sys.modules[__name__]
+        debug_log.setup_logging(into)		# setup global log levels
+        into['cs_debug_logger'] = debug_log.LOGGER
```

Comment:
I like this addition a lot.  I think it's not a bad idea for me to go through the rest of the codebase and add these kinds of debugging statements throughout.

Reply:
The debug logs really helped fix some strange errors during installation - for example, I had to change `MAX_LINE` in `/usr/local/lib/python3.7/site-packages/websockets/http.py` from 4096 to 8192 to get websockets to be happy with the reporter protocol; this wasn't evident until I could see where things were failing.

Reply:
I'm kind of surprised I've never hit that error before.  Is this something to be worried about on a larger scale, or something I should add to the install docs?